mlflow/tracing/export/databricks.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from typing import Sequence
+
+from google.protobuf.json_format import MessageToDict
+from opentelemetry.sdk.trace import ReadableSpan
+from opentelemetry.sdk.trace.export import SpanExporter
+
+from mlflow.entities.trace import Trace
+from mlflow.environment_variables import MLFLOW_HTTP_REQUEST_TIMEOUT
+from mlflow.protos.databricks_trace_server_pb2 import CreateTrace, DatabricksTracingServerService
+from mlflow.tracing.trace_manager import InMemoryTraceManager
+from mlflow.utils.databricks_utils import get_databricks_host_creds
+from mlflow.utils.rest_utils import (
+    _REST_API_PATH_PREFIX,
+    extract_api_info_for_service,
+    http_request,
+)
+
+_logger = logging.getLogger(__name__)
+
+
+_METHOD_TO_INFO = extract_api_info_for_service(
+    DatabricksTracingServerService, _REST_API_PATH_PREFIX
+)
+
+
+class DatabricksSpanExporter(SpanExporter):
+    """
+    An exporter implementation that logs the traces to Databricks Tracing Server.
+    """
+
+    def export(self, root_spans: Sequence[ReadableSpan]):
```

Comment:
```suggestion
    def export(self, spans: Sequence[ReadableSpan]):
```
The export method of SpanExporter should receive non-root spans too. It'd be confusing if we use `root_spans` here.